packages/lesswrong/components/common/AnalyticsTracker.jsx

```diff
@@ -0,0 +1,24 @@
+import { registerComponent } from 'meteor/vulcan:core';
+import React, { useCallback } from 'react';
+import { useTracking } from "../../lib/analyticsEvents";
+
+const buttonDecoding = {
```

Comment:
mouseButton (naively I'd try to map this to "primary/secondary" button, which is a UI thing in material ui)

Reply:
The event returns numbers one through six and I'm mapping roughly accordingly to how the docs for that say they map. It also seems useful to distinguish middle click from right click if I want to. Overall, better not to destroy info by collapsing things.